Board.py

```diff
@@ -0,0 +1,29 @@
+class Board:
+    board: list[list[str]]
+    size: int
+    def __init__(self, size) -> None:
+        self.size = size
+        self.board = self.create_board(size)
+
+    def create_board(self, size) -> list[list[str]]:
+        return [["_" for _ in range(size)] for _ in range(size)]
```

Comment:
```suggestion

    @staticmethod
    def create_board(size: int) -> list[list[str]]:
        return [["_" for _ in range(size)] for _ in range(size)]
```

Methods that don't use any instance/class variables can be made static. See [staticmethod](https://www.tutorialsteacher.com/python/staticmethod-decorator) :)